.github/workflows/build-and-test.yml

```diff
@@ -127,7 +128,10 @@ jobs:
             gradle_version: '8.0'
             java_version: '17'
           - agp_version: '8.2.0'
-            gradle_version: '8.3'
+            gradle_version: '8.2'
```

Comment:
This was a mistake. The min gradle version is 8.2 according to the official document.

![ss 2024-03-08 at 12 37 06](https://github.com/DeployGate/gradle-deploygate-plugin/assets/4340693/3d4fae1b-a0ad-4038-b0d1-8926cb57c333)


https://developer.android.com/build/releases/gradle-plugin